edb/schema/ddl.py

```diff
@@ -397,6 +404,54 @@ def apply_ddl(
     return schema
 
 
+def apply_ddl_script(
+    ddl_text: str,
+    *,
+    schema: s_schema.Schema,
+    modaliases: Optional[Mapping[Optional[str], str]] = None,
+    stdmode: bool = False,
+    testmode: bool = False,
+) -> s_schema.Schema:
+
+    schema, _ = apply_ddl_script_ex(
+        ddl_text,
+        schema=schema,
+        modaliases=modaliases,
+        stdmode=stdmode,
+        testmode=testmode,
+    )
+
+    return schema
+
+
+def apply_ddl_script_ex(
+    ddl_text: str,
+    *,
+    schema: s_schema.Schema,
+    modaliases: Optional[Mapping[Optional[str], str]] = None,
```

Comment:
Probably need to create an alias for `Mapping[Optional[str], str]` (another time, for sure)